app/supportAssignment/directives/supportAssignmentTable/supportCoursesTab/courseHeader/courseHeader.html

```diff
@@ -46,7 +46,7 @@
 					Units
 				</td>
 				<td class="course-header__cell course-header__cell--push-right">
-					{{ sectionGroup.units }}
+					{{ sectionGroup.unitsHigh ? sectionGroup.unitsVariable || sectionGroup.unitsLow + " - " + sectionGroup.unitsHigh : sectionGroup.unitsLow; }}
```

Comment:
We mess up this logic a lot, so I would instead add a transient helper on the entity in the backend so you just see a displayUnits string property which will be the correct thing to show